flipperdevices/Flipper-Android-App
# Add file synchronization

**Background**

Now we upload file from flipper 

**Changes**

- Now we upload/delete file with keys
- New name for Flipper App - Flipper

**Test plan**

- Try open app and synchronizate
- Check that all keys are exist in app
